### PR TITLE
M sc1 59 19

### PR DESCRIPTION
**Pull request by maxharoshka**

### CODE REVIEW

File: mgoroshko/h:w1/python_2.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	<em>
+    <li>Программа на Python — часть 1</li>
+    <li>Программа на Python — часть 2</li>
+    <li>Программа на Python — часть 3</li>
+</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 2</h3>
+<ul>
+	<u>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_2.html]: Inline tag should be inside each li

File: mgoroshko/h:w1/python_2.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 2</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 2</h1>
+<ul>
+	<em>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_2.html]: Inline tag should be inside each li

File: mgoroshko/h:w1/python_2.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+<pre>months=
+['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'] </pre>
+<p>
+Чтобы создать список, мы заключили все значения в квадратные скобки ( '[' и ']' ). Мы
+назвали наш список <em>'months'</em>. Чтобы использовать список, мы дадим команду print
+<strong>months[0]</strong> или <strong>months[1]</strong> (которые выведут на экран <em>'Jan'</em> или <em>'Feb'</em>). Помните: счет
+начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:
+</p>
+<pre>print len(months)</pre>
+<div>которая вернёт 12. </div>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_2.html]: paragraph here

File: mgoroshko/h:w1/python_2.html [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+списков.</p>
+<hr>
+<h2>Комментарии</h2>
+<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.
+Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+Например:</p>
+<p>#</p>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_2.html]: pre here. it is the part of code

File: mgoroshko/h:w1/python_3.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 3</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 3</h1>
+<ul>
+        <em>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_3.html]: Inline tag should be inside each li

File: mgoroshko/h:w1/python_3.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+<li>Программа на Python — часть 1</li>
+<li>Программа на Python — часть 2</li>
+<li>Программа на Python — часть 3</li>
+</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 3</h3>
+<ul><u>
```
Comment (pluhin) [inline mgoroshko/h:w1/python_3.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/Month_of_my_birthday.html [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+		<col width=20>
+		<col width=20>
+		<col width=20>
+		<col width=20>
+		<col width=20>
+		<col width=20>
+	</colgroup>
+	<caption>Месяц моего дня рождения</caption>
+<tr> 
+	<th colspan="6">Calendar</th>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/Month_of_my_birthday.html:19]: Need particular month name

File: mgoroshko/h:w2_Tables_Links/python_1_w_links.html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 1</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 1</h1>
+<ul>
+	<em>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_1_w_links.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/python_2_w_links.html [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 2</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 2</h1>
+<ul>
+	<em>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_2_w_links.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/python_1_w_links.html [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+<li>Программа на Python — часть 1</li>
+<li><a href="python_2_w_links.html">Программа на Python — часть 2</a></li>
+<li><a href="python_3_w_links.html">Программа на Python — часть 3</a></li>
+	</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 1</h3>
+<ul>
+	<u>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_1_w_links.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/python_2_w_links.html [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+	<li><a href="python_1_w_links.html">Программа на Python — часть 1</a></li>
+    <li>Программа на Python — часть 2</li>
+	<li><a href="python_3_w_links.html">Программа на Python — часть 3</a></li>
+</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 2</h3>
+<ul>
+	<u>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_2_w_links.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/python_3_w_links.html [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+<html>
+<head>
+	<META charset="utf-8">
+<title>HOW-TO: Программа на Python, Часть 3</title>
+</head>
+
+<body>
+<h1>HOW-TO: Программа на Python, Часть 3</h1>
+<ul>
+        <em>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_3_w_links.html]: Inline tag should be inside each li

File: mgoroshko/h:w2_Tables_Links/python_3_w_links.html [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+<li><a href="python_1_w_links.html">Программа на Python — часть 1</a></li>
+<li><a href="python_2_w_links.html">Программа на Python — часть 2</a></li>
+<li>Программа на Python — часть 3</li>
+</em>
+</ul>
+<h2>Содержание</h2>
+<h3>HOW-TO: Программа на Python, Часть 3</h3>
+<ul><u>
```
Comment (pluhin) [inline mgoroshko/h:w2_Tables_Links/python_3_w_links.html]: Inline tag should be inside each li

**Review by pluhin — CHANGES_REQUESTED** [review]

File: mgoroshko/h:w2_Tables_Links/python_3_w_links.html (hunk 13 above)
Comment (maxharoshka) [reply mgoroshko/h:w2_Tables_Links/python_3_w_links.html]: fixed everything, can you see the commits?

**Review by pluhin — CHANGES_REQUESTED** [review]
Could you please squash all last commits to one

**Comment by maxharoshka** [comment]
done

**Review by pluhin — APPROVED** [review]
Ok, approved

**Pull request merged**